docs/topics/pagination.txt

```diff
@@ -161,14 +161,14 @@ Optional arguments
 ------------------
 
 ``orphans``
-    The minimum number of items allowed on the last page, defaults to zero.
     Use this when you don't want to have a last page with very few items.
     If the last page would normally have a number of items less than or equal
     to ``orphans``, then those items will be added to the previous page (which
     becomes the last page) instead of leaving the items on a page by
     themselves. For example, with 23 items, ``per_page=10``, and
     ``orphans=3``, there will be two pages; the first page with 10 items and
-    the  second (and last) page with 13 items.
+    the  second (and last) page with 13 items. ``orphans`` defaults to zero,
```

Comment:
Small typo. There is a unnecessary (second) space between two words:
`the  second` -> `the second`.